README.rst

```diff
@@ -100,6 +108,22 @@ LOCK_MODE_WAIT
         0 - DO NOT WAIT, end the operation, and return with error.
         nn - WAIT for nn seconds for the lock to be released.
 
+CONNECTION_RETRY
+    When opening a new connection to the database, automatically retry up to MAX_ATTEMPTS times in
+    the case of errors. Only error codes in ``ERRORS`` will trigger a retry. The wait time between
+    retries is calculated using an exponential backoff with jitter formula::
+
+        random_between(WAIT_MIN, min(WAIT_MAX, WAIT_MULTIPLIER * WAIT_EXP_BASE ** attempt))
+
+    Defaults (wait times are in milliseconds)::
+
+        MAX_ATTEMPTS: 1
+        WAIT_MIN: 0
+        WAIT_MAX: 1000
+        WAIT_MULTIPLIER: 25
+        WAIT_EXP_BASE: 2
+        ERRORS: ['-908', '-27017']
```

Comment:
Really minor but for clarity it might be good to explain what these two are any why they're the default.

Reply:
Done